src/disco/tiles.h

```diff
@@ -112,4 +112,95 @@ typedef struct __attribute__((packed)) {
   uchar  last_entry_hash[32];
 } fd_poh_init_msg_t;
 
+/* A fd_txnm_t is a parsed meta transaction, containing not just the
+   payload */
+
+struct fd_txnm {
+   /* The computed slot that this transaction is referencing, aka. the
+      slot number of the reference_blockhash.  If it could not be
+      determined, this will be a slot around 150 in the future. */
+   ulong    reference_slot;
+
+   ushort   payload_sz;
+
+   /* Can be computed from the txn_t but it's expensive to parse again,
+      so we just store this redundantly. */
+   ulong    txn_t_sz;
```

Comment:
You might want to make this a ushort to make the padding less ugly. That gives you 4 bytes of trailing padding if that could be useful for anything.